core/src/main/resources/reference.conf

```diff
@@ -184,6 +184,8 @@ backend {
         #concurrent-job-limit = 5
 
         run-in-background = true
+        # Configures any logic which should run after the command completes
+        # script-epilogue = ""
```

Comment:
(still ToL) The default is `sync, so I'd put that, commented out. That way, people can see what it is they're changing it away from (assuming they copy/paste from reference.conf of course!).